### ISSUE TITLE
phpmyadmin not working after ubuntu update

### ISSUE THREAD

**Issue by NicolasRannou** [FNNDSC/projectman#71]
Fatal error: require_once(): Failed opening required '/usr/share/php/php-gettext/gettext.inc' (include_path='.') in /usr/share/phpmyadmin/libraries/select_lang.lib.php on line 463

**Comment by rudolphpienaar** [comment]
This is on the "main" chris.tch.harvard.edu, right?

**Comment by NicolasRannou** [comment]
yes.. I looked a bit into it but doesn't really make sense yet...

**Comment by rudolphpienaar** [comment]
Is this still an issue? Nico? Any idea?

**Comment by NicolasRannou** [comment]
Yes I think so, didn't look into it

**Comment by NicolasRannou** [comment]
@BCH and @MGH

**Comment by roc41d** [comment]
try `sudo apt-get install php-gettext` 
works for me +1:

**Comment by furryablack** [comment]
And i have installed php-gettext already =) but still I have the error
